### PR TITLE
Demo GH Flow

### PR DESCRIPTION
**Pull request by rlacquemant**
This pull request aims at demonstrating GH flow. 

- [ ] follow the course
- [ ] do the exercise
- [ ] don't get distracted

### CODE REVIEW

**Comment by g-cheng** [comment]
👎

**Comment by g-cheng** [comment]
@rlacquemant

**Comment by rlacquemant** [comment]
a little fight over here 🔥

File: nightowl_bio.md [hunk 1]
```diff
@@ -0,0 +1 @@
+I usual keep an eye open, anytime.
```
Comment (livlab) [inline nightowl_bio.md]: Wise.

Comment (g-cheng) [inline nightowl_bio.md]: you sure?

**Pull request merged**